test/import_test.erl

```diff
@@ -2,6 +2,9 @@
 
 -include_lib("eunit/include/eunit.hrl").
 
+% call to :module_info
+-elvis([{elvis_style, invalid_dynamic_call, #{ ignore => [{import_test, exports, 1}] }}]).
```

Comment:
Actually, this also requires a recent fix to `elvis_core`, but that one's already merged to `master`.